docs/geedocs/index.html

```diff
@@ -14,7 +14,8 @@
     <p><img src="art/common/googlelogo_color_260x88dp.png" width="130" height="44" alt="Google logo" /></p>
     <h2>Google Earth Enterprise  Documentation</h2>
     <p>Select a Google Earth Enterprise Open Source version to view the documentation</p>
-    <h4><a href="5.2.1/index.html">5.2.1 (unstable)</a></h4>
+    <h4><a href="5.2.2/index.html">5.2.2 (latest stable)</a></h4>
+    <h4><a href="5.2.1/index.html">5.2.1 (stable)</a></h4>
     <h4><a href="5.2.0/index.html">5.2.0 (stable)</a></h4>
   </div>
   <div class="footer">
```

Comment:
Should we add '2017-2018 Open GEE Contributors' ?

Reply:
Since this is a landing page, should we add google analytics to it ? Otherwise we can do that part of another PR.